assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -157,11 +157,7 @@
 
 	.googlesitekit-publisher-win__actions {
 		display: flex;
-		gap: $grid-gap-phone;
-
-		@media (min-width: $bp-desktop) {
-			gap: $grid-gap-desktop;
-		}
+		gap: $grid-gap-phone / 2;
```

Comment:
Same here as above, the change seems correct, but unrelated at first glance. Additional context would be helpful.

Reply:
This uses the same concept as explained in the response in the above comment. This is for the `BannerActions` component.

Before:
![image](https://github.com/google/site-kit-wp/assets/20284937/989de1ec-a44a-4627-8d11-e881b2dd0d03)

After:
![image](https://github.com/google/site-kit-wp/assets/20284937/da9eb002-ab5c-46a8-928d-eda65eec272d)

After (tertiary button hovered):
![image](https://github.com/google/site-kit-wp/assets/20284937/a961306d-6ff5-4c86-a633-0be126b0bf51)